packages/realm/bindgen/CMakeLists.txt

```diff
@@ -24,17 +24,21 @@ if(DEFINED CMAKE_JS_VERSION)
     elseif(CMAKE_HOST_SYSTEM_NAME STREQUAL "Linux")
       if(NODE_ARCH STREQUAL "arm")
         set(CMAKE_TOOLCHAIN_FILE "${CMAKE_CURRENT_SOURCE_DIR}/vendor/realm-core/tools/cmake/armv7-linux-gnueabihf.toolchain.cmake")
+        add_link_options(-fuse-ld=bfd)
         # due to how multiarch works in debian this is needed to link to the correct system libraries
         set(CMAKE_IGNORE_PATH "/usr/lib/x86_64-linux-gnu")
       elseif(NODE_ARCH STREQUAL "arm64")
         set(CMAKE_TOOLCHAIN_FILE "${CMAKE_CURRENT_SOURCE_DIR}/vendor/realm-core/tools/cmake/aarch64-linux-gnu.toolchain.cmake")
         set(CMAKE_IGNORE_PATH "/usr/lib/x86_64-linux-gnu")
+        add_link_options(-fuse-ld=gold)
+        add_link_options(-Wl,-z,noexecstack)
+        add_link_options(-Wl,--warn-execstack)
       elseif(NODE_ARCH STREQUAL "x64")
         set(CMAKE_TOOLCHAIN_FILE "${CMAKE_CURRENT_SOURCE_DIR}/vendor/realm-core/tools/cmake/x86_64-linux-gnu.toolchain.cmake")
+        add_link_options(-fuse-ld=gold)
+        add_link_options(-Wl,-z,noexecstack)
+        add_link_options(-Wl,--warn-execstack)
       endif()
-      add_link_options(-fuse-ld=gold)
-      add_link_options(-Wl,-z,noexecstack)
-      add_link_options(-Wl,--warn-execstack)
```

Comment:
Can you share a bit more context on why this wasn't possible / needed for "arm"?